### PR TITLE
remove the tracing bits

### PR DESCRIPTION
**Pull request by goern**
## This introduces a breaking change

- [x] Yes
- [ ] No

## This Pull Request implements

I have removed the opentracing/jaeger bits.

### CODE REVIEW

**Review by harshad16 — APPROVED** [review]
lgtm, thanks :100:

File: amun/api_v1.py [hunk 1]
```diff
@@ -30,8 +30,8 @@
 from thoth.common import OpenShift
 from thoth.common import datetime2datetime_str
 from thoth.common.exceptions import NotFoundException
-from thoth.storages import InspectionStore
-from thoth.storages.exceptions import NotFoundError as StorageNotFoundError
+from thoth.storages import InspectionStore  # type: ignore
+from thoth.storages.exceptions import NotFoundError as StorageNotFoundError  # type: ignore
```
Comment (fridex) [inline amun/api_v1.py:34]: Why are these `type: ignore` needed?

**Comment by goern** [comment]
ahh... @fridex pre-commit was working locally, but aicoe-ci dropped errors: @harshad16 @KPostOffice 

```
amun/api_v1.py:33: error: Skipping analyzing 'thoth.storages': found module but no type hints or library stubs
amun/api_v1.py:33: note: See https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-imports
amun/api_v1.py:34: error: Skipping analyzing 'thoth.storages.exceptions': found module but no type hints or library stubs
```

**Comment by KPostOffice** [comment]
https://github.com/thoth-station/amun-api/blob/dfe8a51de96bdfadc1c3227d4e38fe91c4b84369/.prow.yaml#L29

If you add `--ignore-missing-imports` here you should be good.

**Comment by goern** [comment]
Can we merge this?

**Pull request merged**